src/extension/statusbartoggler.ts

```diff
@@ -52,14 +65,16 @@ export class StatusBarToggler implements Disposable {
 
     /**
      * update
-     * @description Updates the text displayed in the StatusBarToggler
+     * @description Updates the text and tooltip displayed by the StatusBarToggler
      */
     private update() {
         if (this.isActive) {
             this.statusBarItem.command = StatusBarToggler.removeCommand;
-            this.statusBarItem.text = StatusBarToggler.removeText;
+            this.statusBarItem.tooltip = StatusBarToggler.removeWatchToolTip;
+            this.statusBarItem.text = [this.lineCoverage || "No", StatusBarToggler.coverageText].join(" ");
         } else {
             this.statusBarItem.command = StatusBarToggler.watchCommand;
+            this.statusBarItem.tooltip = StatusBarToggler.watchToolTip;
             this.statusBarItem.text = StatusBarToggler.watchText;
         }
         if (this.isLoading) {
```

Comment:
[Q] When there is a large coverage file we could end up with `$(loading~spin) No Coverage` for a while. Maybe we should change that text to reflect that coverage is being loaded. WDYT?

Reply:
We could have the 4 states as:
1. `Watch` for inactive
2. `$(loading~spin) Coverage` while loading 
3. `$(list-ordered) x% Covered` when actively watching and coverage is available for the active editor
4. `$(list-ordered) No Coverage` when actively watching but no coverage is available or there is no active editor 

Lmk what you think.

Reply:
I went ahead and implemented the above behaviour and separated the code out into a private helper. LMK if you want to see any changes 👍🏻.

Something that I've done / seen elsewhere is the implementing of `$(circle-large-outline)` for "non-loading" state in the UI this fits really well with `$(loading~spin)`. How would you feel about changing `$(list-ordered)` to `$(circle-large-outline)` and having it on all 3 of the non-loading states?

Reply:
Yah I think these changes make a lot of sense especially the large outline change 👍🏻 .